src/applications/covid-research-volunteer/config/form.js

```diff
@@ -0,0 +1,168 @@
+import _ from 'lodash';
```

Comment:
Why is `lodash` added here? I'm not seeing any reference in this file

Reply:
@jhonnyoddball I removed the `lodash` import. Thanks!